WoWsShipBuilder.Web/Components/ShipStatsComponent.razor

```diff
@@ -411,6 +445,14 @@
 
 @code {
 
+    private DepthChargeDataContainer? dcDataContext;
+
+    protected override void OnInitialized()
```

Comment:
This only works for the first selected ship. After switching ships within the same tab, it will never update.
Either don't cache it like this or ensure it is always properly updated.

This component is not recreated when a new ship is selected, only the ViewModel parameter is updated.